### PR TITLE
Localisation updates from https://translatewiki.net.

### PR DESCRIPTION
**Pull request by macgills**

### CODE REVIEW

File: app/src/main/res/values-iw/strings.xml [hunk 1]
```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?><!-- Authors:
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Authors:
 * Amire80
 -->
-<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="TypographyQuotes,TypographyEllipsis">
```
Comment (macgills) [inline app/src/main/res/values-iw/strings.xml]: @amire80 these issue ignorers have been deleted. I guess the only input to TW is the English translation file and if it doesn't have the `ignore` attribute then the generated files won't?
I am speculating about things I really don't know about. I don't know if a ticket needs to be opened TW side to add these to generated Hebrew translation files or if there is anything I can do on my side

Comment (macgills) [reply app/src/main/res/values-iw/strings.xml]: Also @Abijeet further context [here](https://github.com/kiwix/kiwix-android/issues/1711)

TL;DR we don't want to replace straight quotes with curvy quotes or `...` with the ellipsis character in hebrew

File: core/src/main/res/values-qq/strings.xml [hunk 2]
```diff
@@ -18,8 +18,8 @@
   <string name="menu_random_article">{{Identical|Random article}}</string>
```
Comment (macgills) [inline core/src/main/res/values-qq/strings.xml]: Is this metadata file necessary to be added?
If it is we should add `tools:ignore="all"` to it.
If it isn't I would look to delete/stop producing it

**Comment by macgills** [comment]
@Nikerabbit I am not sure if you have free time now? Is there a specific point of contact translatewiki prefer?

**Comment by Abijeet** [comment]
@macgills I'll take a look at the points you've raised tomorrow.

File: app/src/main/res/values-iw/strings.xml (hunk 1 above)
Comment (Abijeet) [reply app/src/main/res/values-iw/strings.xml]: Got it. Currently the Android exporter does not honor attributes added to the resource tag. A ticket needs to be created on our end to preserve such attributes.

For now, would it be possible to create a separate file that defines some configurations for the analysis tool? The way we have `editorconfig` files or `.eslintrc` file?

Comment (macgills) [reply app/src/main/res/values-iw/strings.xml]: we can specify rules for our linter in many places, I chose to put them on these xml files because I wanted them scoped to those files only, as per Amire's request.

In `AllProjectConfigurer` I have
```
lintOptions {
        isAbortOnError = true
        isCheckAllWarnings = true
        isWarningsAsErrors = true

        ignore(
          "SyntheticAccessor",
          //TODO stop ignoring below this
          "MissingTranslation",
          "CheckResult",
          "LabelFor",
          "DuplicateStrings",
          "LogConditional"
        )

        warning(
          "UnknownNullness",
          "SelectableText",
          "IconDensities",
          "ContentDescription",
          "IconDipSize"
        )
        baseline("${path}/lint-baseline.xml")
      }
```
Adding the rule here would disable the check for all files in all modules

Comment (macgills) [reply app/src/main/res/values-iw/strings.xml]: In the interim I can quite easily make these small updates when we do a translation update but if it can be fully automated then that is obviously preferable

Comment (Abijeet) [reply app/src/main/res/values-iw/strings.xml]: Yes, we would like them to be scoped to just that specific file. I assume it's not possible to define config for a particular language file via the `linkOptions` in `AllProjectConfigurer`?

Comment (macgills) [reply app/src/main/res/values-iw/strings.xml]: I can point to a `lint.xml` like [this](https://developer.android.com/studio/write/lint#sample-lint.xml-file) to specify paths to ignore, the option is not present in the `lintOptions` closure of our kotlin build plugin and I was trying to avoid spreading our lint configuration across multiple files but it seems the most viable option

File: core/src/main/res/values-qq/strings.xml (hunk 2 above)
Comment (Abijeet) [reply core/src/main/res/values-qq/strings.xml]: Its highly recommended that the qqq file is retained. This is the message documentation file. 

You can read more about here - https://translatewiki.net/wiki/Translating:Localisation_for_developers#Message_documentation

The idea is that when you add a new string in English, you add some documentation in qq file to provide translators some context regarding the string.

I think we can add `tools:ignore="all"` part to the resources tag, will discuss with Niklas.

File: app/src/main/res/values-iw/strings.xml (hunk 1 above)
Comment (Abijeet) [reply app/src/main/res/values-iw/strings.xml]: Yes, until we fix this on our end.

File: core/src/main/res/values-qq/strings.xml (hunk 2 above)
Comment (macgills) [reply core/src/main/res/values-qq/strings.xml]: Then I should document that on our side. I will create a ticket

Comment (macgills) [reply core/src/main/res/values-qq/strings.xml]: https://github.com/kiwix/kiwix-android/issues/1759 ticket added

File: core/src/main/res/values-eu/strings.xml [hunk 3]
```diff
@@ -41,7 +41,7 @@
   <string name="pref_back_to_top">Gora itzuli</string>
   <string name="pref_back_to_top_summary">Bistaratu botoi bat orrialdearen amaieran goiko aldera mugitzeko</string>
   <string name="pref_hide_toolbar">Tresna-barra ezkutatu desplazatzean</string>
-  <string name="pref_hide_toolbar_summary">\u2022 On GNU/Linux and with the console: split —bytes=4000M my_big_file.zim</string>
+  <string name="pref_hide_toolbar_summary">u2022 On GNU/Linux and with the console: split —bytes=4000M my_big_file.zim</string>
```
Comment (macgills) [inline core/src/main/res/values-eu/strings.xml]: the English source for this is
```
Keeps the toolbar from scrolling out of view when browsing content
```
and removing the `\` here stops it becoming a Unicode character, this will just display `u2022`

Comment (macgills) [reply core/src/main/res/values-eu/strings.xml]: @Abijeet

File: app/src/main/res/values-iw/strings.xml (hunk 1 above)
Comment (Abijeet) [reply app/src/main/res/values-iw/strings.xml]: Added [T244494](https://phabricator.wikimedia.org/T244494) on our end.

File: core/src/main/res/values-eu/strings.xml (hunk 3 above)
Comment (Abijeet) [reply core/src/main/res/values-eu/strings.xml]: Hmm, this looks like this is an issue on our end. This was removed during an import - https://translatewiki.net/w/i.php?title=Kiwix%3AAndroid.ui.pref_hide_toolbar_summary%2Feu&type=revision&diff=9243340&oldid=9108321 after this should change was made - https://github.com/kiwix/kiwix-android/commit/581b127184259ae999bd3501d95bb908aaffa1e5#diff-c533bde4f259f63373679ba0e169ac32R44

This is obviously wrong and needs some investigation.

Comment (macgills) [reply core/src/main/res/values-eu/strings.xml]: In the meantime I am pretty happy to merge this if you want to approve the PR.

While this string is in error I don't think we have enough basque users to cause confusion

Comment (Abijeet) [reply core/src/main/res/values-eu/strings.xml]: Created [T244501](https://phabricator.wikimedia.org/T244501) to track this. I'll resolve this, and approve the PR.

Thanks for reporting these issues, and for your patience.

Comment (Abijeet) [reply core/src/main/res/values-eu/strings.xml]: I've fixed this translation. This should be exported during the next export on Monday, 10th Feb.
